pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -66,6 +67,11 @@ const (
 var validInterfaceModels = map[string]*struct{}{"e1000": nil, "e1000e": nil, "ne2k_pci": nil, "pcnet": nil, "rtl8139": nil, "virtio": nil}
 var validIOThreadsPolicies = []v1.IOThreadsPolicy{v1.IOThreadsPolicyShared, v1.IOThreadsPolicyAuto}
 var validCPUFeaturePolicies = map[string]*struct{}{"": nil, "force": nil, "require": nil, "optional": nil, "disable": nil, "forbid": nil}
+var ValidateNamespaceName = apimachineryvalidation.ValidateNamespaceName
```

Comment:
does this block repesent the modifacations in the original file needed or is this an artifact of the original PR? Would you mind doing a clear copy and commit of the original file, just with the header info attached, and do the modification in a followup one? It makes it easier to see what we had to modify when updating the file.

Reply:
updated